.github/actions/version_checking/action.yml

```diff
@@ -0,0 +1,36 @@
+name: version_checking
+description: 'Сhecks the equality of version from code and the last repository tag'
+
+runs:
+  using: "composite"
+  steps:
+    - name: Clone the module
+      uses: actions/checkout@v2
+
+    - name: Setup tarantool
+      uses: tarantool/setup-tarantool@v1
+      with:
+        tarantool-version: '2.9'
```

Comment:
You use old code here without our review fixes

Reply:
Now 2.10